testing/marketplace/deployer/helm_tiller_onbuild/standard/Dockerfile

```diff
@@ -0,0 +1,3 @@
+ARG MARKETPLACE_TOOLS_TAG
```

Comment:
It's unfortunate that we have to have 2 separate similar sets of folders to test a rather small feature of the onbuild version. Can you follow up to clean this up? I think these should just be 2 test cases with 2 separate entry scripts.

Reply:
Certainly will follow up, though it's not clear to me what "clean" looks like. Let's discuss. :)